### ISSUE TITLE
Suguestion: flag for default UID when none are aliased like groups

### ISSUE THREAD

**Issue by WobbleTheHutt** [maharmstone/btrfs#496]
Just got everything working on my steam deck with shared steam library folders but for some reason steam writes a user ID of 65536. my UID is aliased and works fine for file creation. My current solution is chmod and chowning the files on boot to steam OS but it's kinda an ugly hack.

**Comment by kinemitor** [comment]
Came here looking for how to set de default user and group for the same reasons.
Steam deck uses 1000 for the user ID.
There is currently no way to make it the default and changing a folder in the properties menu is not recursive.

**Comment by maharmstone** [comment]
Have a look at the Mappings section in the README. I do plan a GUI for this as well at some point.

**Comment by WobbleTheHutt** [comment]
Yeah I was looking at mappings but could not figure out the process or
account calling it I tried mapping everything including trusted installer.
Any way to just have a custom default if not specified?

**Comment by kinemitor** [comment]
thx, I figued it out.
the document talk about "Windows SID" but did not say how to get it, what I did was to open HKEY_USERS and because there is only one user the SID was obvious.
Now to set the id in HKLM\SYSTEM\CurrentControlSet\services\btrfs\Mappings you need to select Decimal in the dword edit.

**Comment by WobbleTheHutt** [comment]
thanks ill test it out this weekend gotta get dualboot setup again

**Comment by mio-19** [comment]
+1
